src/Pages/Profile.js

```diff
@@ -1,13 +1,86 @@
-import { Aside, NavBar } from "../components"
+import { useState } from "react"
+import { useSelector , useDispatch} from "react-redux"
+import { OpenEditProfileModal , CloseEditProfileModal} from "../features/modal/modalSlice"
+import { Aside, Modal, NavBar, SinglePost } from "../components"
+import { editUserProfile } from "../features/auth/authSlice"
 import { MainContainer , Feed} from "../styled.components"
-
+import { PrimaryStyledButton } from "../styled.components/Button"
+import { StyledProfileWrapper } from "../styled.components/ProfileWrapper"
+import { StyledTextAreaWithBorder } from "../styled.components/TextArea"
 
```

Comment:
suggestion: information that needs to be added like followers and following count, total posts